etc/helpers.js

```diff
@@ -9,6 +9,8 @@ import cookies from "js-cookie";
 
 export const usedLangIds = ["en", "cz", "ro", "hu", "es", "fr", "de", "ru", "it"];
 
+export const UTM_PARAMS = process.env.UTM_PARAMS ? `/?${process.env.UTM_PARAMS}` : "";
```

Comment:
it should be default "/" then. why not to put "/" directly to links? it is not a part of params anyway